pkg/autoupdate/autoupdate.go

```diff
@@ -43,15 +43,41 @@ const (
 
 // Map for easy lookups of supported policies.
 var supportedPolicies = map[string]Policy{
-	"":         PolicyDefault,
-	"disabled": PolicyDefault,
-	"image":    PolicyRegistryImage,
-	"registry": PolicyRegistryImage,
-	"local":    PolicyLocalImage,
+	"":                          PolicyDefault,
+	string(PolicyDefault):       PolicyDefault,
+	"image":                     PolicyRegistryImage,
```

Comment:
I think `image` is deprecated, right? Could you add note here if this is the case.

Reply:
Very good though, done!